packages/demand-deeplink/src/validate.ts

```diff
@@ -109,10 +112,26 @@ function validatePickupTime(time: string | undefined, bookingType: BookingType)
   return errors
 }
 
+function isPositionHasUndefinedCoordinates(position: Position): boolean {
+  const { lat, lng } = position
+
+  return isUndefined(lat) && isUndefined(lng)
+}
+
+function validatePosition(position: Position, path: string) {
+  const { lat, lng } = position
+
+  return (lat && !latitudeRegexp.test(lat)) || (lng && !longitudeRegexp.test(lng)) || !lat || !lng
```

Comment:
I guess the condition should be like: `(lat && !latitudeRegexp.test(lat)) && (lng && !longitudeRegexp.test(lng))`